radio/src/opentx.h

```diff
@@ -462,14 +462,19 @@ extern void boardInit();
 #endif
 
 #if defined(PCBTARANIS)
+ #if defined(TARANIS_INTERNAL_PPM)
+  #define IS_MODULE_PPM(idx)                (idx==TRAINER_MODULE || (idx==EXTERNAL_MODULE && g_model.externalModule==MODULE_TYPE_PPM)|| (idx==INTERNAL_MODULE && g_model.internalModule==MODULE_TYPE_PPM))
+  #define IS_MODULE_XJT(idx)                (((idx==INTERNAL_MODULE && g_model.internalModule==MODULE_TYPE_XJT)|| (idx==EXTERNAL_MODULE && g_model.externalModule==MODULE_TYPE_XJT)) && (g_model.moduleData[idx].rfProtocol != RF_PROTO_OFF))
+  #else
   #define IS_MODULE_PPM(idx)                (idx==TRAINER_MODULE || (idx==EXTERNAL_MODULE && g_model.externalModule==MODULE_TYPE_PPM))
-  #define IS_MODULE_XJT(idx)                ((idx==INTERNAL_MODULE || g_model.externalModule==MODULE_TYPE_XJT) && (g_model.moduleData[idx].rfProtocol != RF_PROTO_OFF))
+  #define IS_MODULE_XJT(idx)                (((idx==INTERNAL_MODULE && g_model.internalModule==MODULE_TYPE_XJT)|| (idx==EXTERNAL_MODULE && g_model.externalModule==MODULE_TYPE_XJT)) && (g_model.moduleData[idx].rfProtocol != RF_PROTO_OFF))
```

Comment:
It is used in menu_model_setup.cpp (MENU_TAB) and internal module doesn't necessarly mean XJT anymore as before. Or did I miss something ?